docs/product/issues/issue-priority/index.mdx

```diff
@@ -36,9 +36,9 @@ You can update issue priority manually at any time, either from the issue detail
 
 ## Priority in the Issue Stream
 
-By default, the issue stream on the **Issues** page filters out low-priority issues. If you'd like to be able to see low-priority issues, you can [set your default search](/product/reference/search/saved-searches/#set-a-default-search) to include them.
+The default view on the **Issues** page is the "Prioritized" view, which shows high and medium priority issues. But all views can be filtered to include, for example, only high priority issues. You can do this by adding the query: `issue.priority:high` to the search bar and saving the view.
```

Comment:
I'd try to retain indicating here that low prio can still be searched for, which feels like what the original text was going for. Maybe changing your example to `issue.priority:low`instead.